lib/mp4/transmuxer.js

```diff
@@ -862,6 +864,14 @@ CoalesceStream.prototype.flush = function(flushSource) {
     timelineStartPts = this.videoTrack.timelineStartInfo.pts;
   } else if (this.audioTrack) {
     timelineStartPts = this.audioTrack.timelineStartInfo.pts;
+    ['audioobjecttype',
```

Comment:
I would love it if we could camelCase these names but that would be a more major change so I'm just complaining and don't expect you to change that for this PR